docs/source/developer_documentation.rst

```diff
@@ -0,0 +1,131 @@
+
+
+New module or classes
+~~~~~~~~~~~~~~~~~~~~~
+
+For example, under the ``./docs/source/msei_reference``, the ``index.rst`` file lists the ``HospitalAdmissions`` model by adding the following entry:
+
+.. code-block:: RST
+
+  Hospital Admissions
+  -------------------
+
+  .. automodule:: pyrenew.latent.hospitaladmissions
+     :members:
+     :undoc-members:
+     :show-inheritance:
+
+This entry tells Sphinx to generate documentation for the ``HospitalAdmissions`` model and its members.
+
+New tutorials
+~~~~~~~~~~~~~
+
+``PyRenew`` tutorials are `quarto documents <https://quarto.org>`__ located under `./model/docs <https://github.com/CDCgov/multisignal-epi-inference/tree/main/model/docs>`__. Tutorials are automatically rendered using GitHub actions (see the Workflow file `here <https://github.com/CDCgov/multisignal-epi-inference/actions/workflows/website.yaml>`__). To make the new tutorial available in the website, developers should follow these steps:
+
+1. Create a new ``quarto`` file in the ``./model/docs`` directory. For instance, the ``example_with_datasets.qmd`` file was added to the repository.
+
+2. Add an entry in the ``./docs/source/tutorials/index.rst``, for example:
+
+.. code-block:: RST
+
+   .. toctree::
+      :maxdepth: 2
+          getting_started
+          example_with_datasets
+
+3. Add an rst entry with the same basename as the ``quarto`` file in the ``./docs/source/tutorials`` directory. For instance, the `example_with_datasets.rst` file was added to the repository. This last step can be done running the bash script `./.pre-commit-rst-placeholder.sh <https://github.com/CDCgov/multisignal-epi-inference/blob/main/.pre-commit-rst-placeholder.sh>`__. Note the script should be executed by ``pre-commit``.
+
+
+Adding new pages
+~~~~~~~~~~~~~~~~
+
+Sphinx also allows adding arbitrary pages. For instance, all the ``PyRenew`` tutorials are additional documentation. The steps to add a new page are:
+
+1. Create a ``rst`` file in the appropriate directory. For example, this file about development was added under ``./docs/source/developer_documentation.rst``.
+
+
+2. Make sure the new ``rst`` file is included in an indexed file, for instance, ``./docs/source/general/ctoc.rst``. Here is how it looks:
+
+.. code-block::
+
+   Complete Table Of Contents
+   ==========================
+
+   .. toctree::
+      :maxdepth: 2
+
+       ../index
+       ../msei_reference/index
+       ../tutorials/index
+       ../genindex
+       ../developer_documentation
+
+
+The last entry is the ``developer_documentation`` page.
+
+Additional Developer Info
+-------------------------
+
+-  makefile
+-  numpydocs
+-  How does typos work?
+-  download artifact for website preview
+-  How to create a new tutorial
+-  How to add documentation for a new module to the website
```

Comment:
Perhaps this can be removed from this PR in a first pass and this TODO list can be placed in #242 ? Same with Background Information.

Reply:
addressed in https://github.com/CDCgov/multisignal-epi-inference/pull/224/commits/d8b6131d85d92347ebae0dde6c62e1043465eac9